src/comments.js

```diff
@@ -575,6 +576,16 @@ function handlePropertyComments(enclosingNode, comment) {
   return false;
 }
 
+function handleExportNamedDeclarationComments(enclosingNode, comment) {
+  if (enclosingNode && enclosingNode.type === "ExportNamedDeclaration") {
+    // Enforce all comments to be leading block comments.
+    comment.type = "CommentBlock";
```

Comment:
Ok, I was not sure about that one and I found it weird at first glance when I tried it.